ctru-sys/src/bin/docstring-to-rustdoc.rs

```diff
@@ -0,0 +1,70 @@
+//! This script transforms _some_ Boxygen comments to Rustdoc format
```

Comment:
Nice! I had been looking for something like this builtin to `bindgen` but this little script is a good workaround.